.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
This is a bit over-general, like "How to solve the problem. Step 1) describe the solution to the problem. Step 2) ...". Maybe "What went wrong?".

Reply:
How about "Describe your expectation and what Cirq does instead"?